### PR TITLE
Add data manager for mash sketches

### PR DESCRIPTION
**Pull request by dfornika**
FOR CONTRIBUTOR:
* [x] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [x] - License permits unrestricted use (educational + commercial)
* [x] - This PR adds a new tool or tool collection
* [ ] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

### CODE REVIEW

File: tools/mash/mash_screen.xml [hunk 1]
```diff
@@ -52,18 +69,21 @@
...
             <output name="output" file="mash_screen_NZ_MYON01000010.1_ERR024951_seqtk_sample_1000_1.tsv"/>
         </test>
         <test>
+            <param name="queries_input_source_selector" value="history"/>
             <param name="queries" value="NZ_MYON01000010.1.msh"/>
             <param name="pool_input_selector" value="single"/>
             <param name="pool" value="ERR024951_seqtk_sample_1000_2.fastq"/>
             <output name="output" file="mash_screen_NZ_MYON01000010.1_ERR024951_seqtk_sample_1000_2.tsv"/>
         </test>
         <test>
```
Comment (bgruening) [inline tools/mash/mash_screen.xml]: can we convert one of the tests to use the loc file?

**Comment by bgruening** [comment]
@dfornika thanks a lot.

**Comment by dfornika** [comment]
@bgruening Thanks for merging this. Unfortunately the data manager didn't seem to make it into the toolshed. Is there any way to push it manually, or could you suggest a way to trigger a new deployment?

**Comment by bgruening** [comment]
Yes, pushing manually. Sorry for the delay.

**Comment by dfornika** [comment]
@bgruening Thanks! No apologies necessary. Honestly I don't know how you keep on top of all this stuff so quicky. It's amazing.

**Pull request merged**